rep-0147.rst

```diff
@@ -221,7 +223,7 @@ The reference position is the origin of the ``base_link`` as defined in [#REP105
 .. They are well established and have several existing implementations.
 .. The ``mav_msgs/RollPitchYawrateThrust`` is a convenience message to support low level controllers that are not aware of the absolute yaw.
```

Comment:
Should these messages not still be standardized? Variations on AttitudeThrust and RollPitchYawrateThrust are often used to communicate between a high-level position/velocity controller, and a low-level attitude controller.

Reply:
I would suggest that the existing defined interfaces are 'High-level interfaces', and 'Low-level interfaces' are the two referenced above.